Embedded Software/TelemetryControlUnit/include/xbee_at.h

```diff
@@ -0,0 +1,65 @@
+/**
+ * @file xbee_at.h
+ * @author Peter Rydzynski
+ * @brief Utilities to send and receive data with the Xbee wireless modules
+ * @date 2022-05-16
+ * 
+ * @details Xbees have two device modes, API and AT. This file defines AT mode, which
+ * acts as a raw serial bridge between the two communicating modules. API mode, on
+ * the other hand, defines packet structures for more functionality (and more complexity).
+ * See the teensy 4.1 docs for an impelementation of API mode.
+ *
+ */
+
+#include <Arduino.h>
+
+/* Error codes for the function return values */
+#define XBEE_SUCCESS              0
+#define XBEE_ERROR_RX_EMPTY       1
+#define XBEE_ERROR_TX_FULL        2
+#define XBEE_ERROR_MAX_CALLBACKS  3
+#define XBEE_ERROR_RECEIVE_MODE   4
+#define XBEE_ERROR_RECEIVE_FORMAT 5
+#define XBEE_ERROR_CONNECTION     6
```

Comment:
Might be better to define these as static const enums, or create a new "xbeeError_t" data type enum

Reply:
But also we might not use xbees going forward so like oop

Reply:
I want big antenna >:)

Reply:
Will fix this soon